### DISCUSSION TITLE
Keypad - HA

### DISCUSSION THREAD

**Discussion by wifispray** [tsightler/ring-mqtt#439] [Support Q&A]
Hi All.

Im new here so sorry for the basic questions. I can see the ability to see the keypad, is there a way to input the alarm code? Im thinking of having a tablet near the backdoor.

**Comment by tsightler** [comment]
There is no method to automatically use the same codes as the alarm panel.  You can define a single code to disarm the alarm which is used by the HA GUI or you can do more advanced things with HA itself.  Basically, the virtual keypad in HA is completely separate from the Ring keypad and has it's own codes.

**Comment by wifispray** [comment]
Thanks. How do you do this?

**Reply by tsightler** [reply]
Set the disarm code in the configuration, that's it.
